features/support/ui/your_contributions.rb

```diff
@@ -8,6 +8,11 @@ class YourContributions < SitePrism::Page
     set_url '{/locale}/tools/wpcc/your_contributions'
     set_url_matcher(/wpcc\/\d+\/your_contributions/)
 
+    section :employee_percent,   EmployeePercentSection, '.contributions__source--employee'
+    section :employer_percent,   EmployerPercentSection, '.contributions__source--employer'
+    
     element :details, '.details__row'
+    element :contributions_description, '.t-contributions_description'
```

Comment:
We are partially following the usage of the `.t-` standard for page elements without a unique natural identifier. 

There is no real issue of coupling here. Using `.t-` informs anyone who seeks to make alterations to the view that this change could possibly break a test.

This is a good condition for making it absolutely clear which sections and elements of the view are under tests. 

@tomas-stefano @margOnline @willhall88 I think that it should be permissible to use `.t-` in conjunction with other ways of targeting page elements and sections. 

What are your opinions on this @davidtrussler @johnplayer1982 @thecountgs?